ngboost/distns/poisson.py

```diff
@@ -43,15 +39,15 @@ def __init__(self, params):
         self.dist = dist(mu=self.mu)
 
     def fit(Y):
-        assert(np.equal(np.mod(y, 1), 0).all), "All Poisson target data must be discrete integers"
+        assert(np.equal(np.mod(Y, 1), 0).all), "All Poisson target data must be discrete integers"
         assert(np.all([y >= 0 for y in Y])), "Count data must be >= 0"
```

Comment:
```suggestion
        assert((y>=0).all()), "Count data must be >= 0"
```
Removing list comprehension for speed.

side note: I'm happy to take input on `np.all(some_array)` vs. `some_array.all()` . The latter is marginally faster (I believe because it skips a series of checks that see if the input needs to be converted to an np array. so you either save time on the check or the conversion. Regardless, the time saved here is trivial.) but can be considered less readable.